components/central-application-gateway/README.md

```diff
@@ -0,0 +1,109 @@
+
+## Prerequisites
+
+The Central Application Gateway requires Go 1.8 or higher.
+
+## Installation
+
+To install the Central Application Gateway, follow these steps:
+
+1. `git clone git@github.com:kyma-project/kyma.git`
+2. `cd kyma/components/central-application-gateway`
+3. `CGO_ENABLED=0 go build ./cmd/applicationgateway`
+
+## Usage
+
+This section explains how to use the Central Application Gateway.
+
+### Start the Central Application Gateway
+
+To start the Central Application Gateway, run this command:
+
+```
+./applicationgateway
+```
+
+The Central Application Gateway has the following parameters:
+- **disableLegacyConnectivity** is the flag for disabling the default legacy and enabling the Compass mode. The default value is `false`.
+- **proxyPort** is the port that acts as a proxy for the calls from services and Functions to an external solution. The default port is `8080`.
+- **externalAPIPort** is the port that exposes the API allowing to check component status. The default port is `8081`.
+- **application** is the Application name used to write and read information about services. The default Application is `default-ec`.
+- **namespace** is the Namespace in which the Central Application Gateway is deployed. The default Namespace is `kyma-system`.
+- **requestTimeout** is the timeout for requests sent through the Central Application Gateway, expressed in seconds. The default value is `1`.
+- **skipVerify** is the flag for skipping the verification of certificates for the proxy targets. The default value is `false`.
+- **requestLogging** is the flag for logging incoming requests. The default value is `false`.
+- **proxyTimeout** is the timeout for requests sent through the proxy, expressed in seconds. The default value is `10`.
+- **proxyCacheTTL** is the time to live of the remote API information stored in the proxy cache, expressed in seconds. The default value is `120`.
+
+
+## API
+The Central Application Gateway exposes:
+- external API implementing health endpoint for liveness and readiness probes
+- internal API implementing proxy handler accessible via a service of ClusterIP type
+
+### Standalone mode
+In case  **disableLegacyConnectivity** is `false` the proxy API exposes the following endpoint:
+```{Application name}/{Service name}/{target API path}``` 
+
+For instance, if the user registered `cc-occ-commerce-webservices` service in `ec` application using Application Registry, they can send a request to the following url:
+```http://central-application-gateway:8080/ec/cc-occ-commerce-webservices/basesites```
+
+As a result Central Application Gateway performs the following steps:
+- looks for `cc-occ-commerce-webservices` service in `ec` Application CRD and extracts target URL path along with authentication configuration
+- modifies request to include authentication data
+- sends request to the following path `{target URL extracted from the Application CRD}/basesites`
```

Comment:
``````suggestion
As a result, the Central Application Gateway:
- looks for the `cc-occ-commerce-webservices` service in the `ec` Application CRD and extracts the target URL path along with the authentication configuration
- modifies the request to include the authentication data
- sends the request to the following path:
   ```bash
   {TARGET_URL_EXTRACTED_FROM_APPLICATION_CRD}/basesites
   ```
``````
I added the definite article `the` in front of certain nouns where it was missing, and shortened the first sentence, 'cause it didn't match the list items grammatically (they would have to be `looking`, `modifying`, `sending`, but `looks`, `modifies` and `sends` reads better; this way, they can stay as they are). I also fixed the path formatting.

We should be consistent with the definite article `the`. We plan to drop the articles in front of our components, so it's okay to drop it, but then we should drop it throughout the whole document. 
You use `the` in front of `Central Application Gateway` in the rest of this document, so I added it here as well. 
<br>

Also, if the order of these steps matter, we should format them as an ordered list (with numbers). ❗